critiquebrainz/data/model/user.py

```diff
@@ -95,9 +95,9 @@ def reviews(self):
     def avatar(self):
         """Link to user's avatar image."""
         if self.show_gravatar and self.email:
-            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=mm&r=pg"
+            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=identicon&r=pg"
         else:
-            return "https://gravatar.com/avatar/placeholder?d=mm"
+            return "https://gravatar.com/avatar/placeholder?d=identicon"
```

Comment:
It works fine alright. I just tested it on mine and yours user id.
This means each user will now have their own gravatar incase they don't have one associated to their email and will get a unique one for them only till the time their user id remains the same (which will)